docker/run

```diff
@@ -2,6 +2,8 @@
 
 set -e
 
+/etc/init.d/clamav-daemon start
```

Comment:
Has that slipped in from another PR?

Reply:
Kind of, I was trying to fix a bug in which `clamav-daemon` doesn't restart  when the container restart. I could write it in another PR

Reply:
No problem - just making sure it was something that would cause a error on its own.